README.md

```diff
@@ -168,6 +170,20 @@ See below:
 - Search for their historical git by secrets and other contents exposed;
 - Your analysis can be fully configurable, [**see all CLI available resources**](https://horusec.io/docs/cli/resources/#3-flags).
 
+## **Documentation**
+You can find Horusec's documentation on our [**website**](https://horusec.io/docs/).
+
+## **Roadmap**
+We have a project [**roadmap**](ROADMAP.md), you can come contribute with us!
+
+Horusec also have other repositories, check out some of our other projects:
+
+- [**Horusec Platform**](https://github.com/ZupIT/horusec-platform)
+- [**Horusec DevKit**](https://github.com/ZupIT/horusec-devkit)
+- [**Horusec Engine**](https://github.com/ZupIT/horusec-engine)
+- [**Horusec Operator**](https://github.com/ZupIT/horusec-operator)
+- [**Horusec Admin**](https://github.com/ZupIT/horusec-admin)
```

Comment:
Since admin is deprecated, should we keep it? 

@wiliansilvazup

Reply:
We can remove 
```suggestion
```